### PR TITLE
Refact/react app

### PR DESCRIPTION
**Pull request by felipebrunetti94**

### CODE REVIEW

File: random-circle-generator/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      }
+		</style>
+	</head>
+
+  <body style="background:rgb(163, 163, 163)">  
+    
+    <div id="root"></div>				
+    
+	</body>
+</html>
```
Comment (negreirosleo) [inline random-circle-generator/index.html]: Any specific reason for that indentation? :eyes:

File: random-circle-generator/script2.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  stop = document.getElementById('stop'),
+  slow = document.getElementById('slowSpeed'),
+  normal = document.getElementById('normalSpeed'),
+  fast = document.getElementById('fastSpeed'),
+  showCount = document.getElementById('count'),
+  Speed = 5000;
+//not working change speed generate circle functions!!!!
+
+function setSpeed(sp){
+    Speed = sp
```
Comment (negreirosleo) [inline random-circle-generator/script2.js]: how about 2 spaces indentaion here?

File: random-circle-generator/script2.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+const makeCircles = () => {
+    var y = Math.floor((Math.random() * (screen.height - 200)) + 40).toString();
+    var x = Math.floor((Math.random() * (screen.width - 200)) + 1).toString();
+    var radius= Math.floor((Math.random() * 200) + 1).toString();
+    var colorNumber=Math.floor((Math.random()*10)+0)
+    showCount.textContent = count;
+    createCircle(x,y,radius,colorNumber)
+    setTimeout(makeCircles, Speed)
+    console.log("i did a circle")
```
Comment (wps13) [inline random-circle-generator/script2.js]: Forgot console here

File: random-circle-generator/script2.js [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+})
+
+
+const createSquares = () => {
+
+}
+
+const removeCircles = () => {
+
+}
```
Comment (negreirosleo) [inline random-circle-generator/script2.js]: Are these doing anything?

File: random-circle-generator/script2.js [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+const speedUp = () => {
+  Speed = Speed / 2
+  updateSpeed()
+}
+
+const updateSpeed = () => {
+  let speedInSeconds = Speed/1000
+  let updatedSpeed = 1/speedInSeconds + "circles/second"
+  document.getElementById("actualSpeed").textContent = updatedSpeed
+  console.log(updatedSpeed)
```
Comment (wps13) [inline random-circle-generator/script2.js]: Another console here

File: random-circle-generator/src/components/frame.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    left: style.left,
+    bottom: style.bottom,
+    width: style.size,
+    height: style.size,
+    borderWidth: 8,
+    borderStyle: 'solid',
+    borderColor: style.color,
+    borderRadius: '50%'
+  }
+  console.log("style")
```
Comment (wps13) [inline random-circle-generator/src/components/frame.js]: More consoles here.

File: random-circle-generator/src/components/nav.js [hunk 7]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import Button from './button'
+
+
+
+let ulStyle = {
```
Comment (wps13) [inline random-circle-generator/src/components/nav.js]: Any specific reason for mixing inline style with external css?

File: random-circle-generator/index.html (hunk 1 above)
Comment (iagodahlem) [reply random-circle-generator/index.html]: You can use [EditorConfig](https://editorconfig.org/) to help you with that. It helps a lot to keep your code style consistent between everyone working on the same project. You just have to install it on your editor and add a `.editorconfig` file, the most commonly used seems like this:

```
# EditorConfig helps developers define and maintain consistent
# coding styles between different editors and IDEs
# http://editorconfig.org

# top-most EditorConfig file
root = true

[*]
charset = utf-8
end_of_line = lf
indent_style = space
indent_size = 2
trim_trailing_whitespace = true
insert_final_newline = true
```

**Pull request merged**